spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/properties/KafkaConsumerProperties.java

```diff
@@ -253,12 +257,32 @@ public void setDestinationIsPattern(boolean destinationIsPattern) {
 		this.destinationIsPattern = destinationIsPattern;
 	}
 
+	/**
+	 * No longer used; get properties such as this via {@link #getTopic()}.
+	 * @deprecated No longer used
+	 */
+	@Deprecated
+	@DeprecatedConfigurationProperty(reason = "Not used since 2.1.1, set properties such as this via 'topic'")
 	public KafkaAdminProperties getAdmin() {
 		return this.admin;
 	}
 
 	public void setAdmin(KafkaAdminProperties admin) {
```

Comment:
We need `@SuppressWarnings("deprecation")` on these accessors.

Reply:
done!

Reply:
Needed on the setter too.